docs/hopsworksai/azure/cluster_creation.md

```diff
@@ -69,7 +74,23 @@ In order to let the cluster instances access to the Azure storage we need to att
   </figure>
 </p>
 
-### Step 5 Virtual network selection
+### Step 5 set the backup retention policy:
+
+!!! note
+    This step is only accessible to enterprise users.
+
+To back up the Azure blob storage data when taking a cluster backups we need to set a retention policy for the blob storage. In this step, you choose the length in days of this data retention. You can deactivate the retention policy by setting this value to 0 but this will block you from taking any backup of your cluster.
```

Comment:
```suggestion
To back up the Azure blob storage data when taking a cluster backups we need to set a retention policy for the blob storage. In this step, you choose the retention period in days. You can deactivate the retention policy by setting this value to 0 but this will block you from taking any backup of your cluster.
```